### ISSUE TITLE
CouchDB: Only update data that has changed in PostgreSQL

### ISSUE THREAD

**Issue by perlun** [perlun/pg2couch#3]
Records where the CouchDB representation is identical to the data in the source (Postgres) must not be inserted, since we will create a huge number of CouchDB document revisions if we use it that way.

**Comment by krainboltgreene** [comment]
Is it feasible to create a checksum of the record before shoving it to couchdb?

**Comment by perlun** [comment]
@krainboltgreene 

> Is it feasible to create a checksum of the record before shoving it to couchdb?

Yep, it should be.  Something like [this](http://csm.tumblr.com/post/18963100318/how-couchdb-revision-number-generation-works) - I think I had a better link at some point, maybe in the CouchDB or PouchDB documentation somewhere.
